quarkus-workshop-super-heroes/super-heroes/extension-version/deployment/src/main/java/io/quarkus/workshop/superheroes/version/deployment/ExtensionVersionProcessor.java

```diff
@@ -19,7 +20,7 @@ FeatureBuildItem feature() {
     @BuildStep
     @Record(ExecutionTime.RUNTIME_INIT)
     void recordVersion(ApplicationInfoBuildItem app, VersionConfig versionConfig, VersionRecorder recorder) {
-        if (versionConfig.enabled) {
+        if (versionConfig.enabled()) {
```

Comment:
If the config is runtime, this shouldn't be done in the processor

Reply:
Just to confirm: should we pass `VersionConfig` to the `io.quarkus.workshop.superheroes.version.runtime.VersionRecorder#printVersion` and call the `versionConfig.enabled()` there?